dist/obs-bundled-gems.spec

```diff
@@ -142,8 +142,10 @@ find %{buildroot} -type f -print0 | xargs -0 grep -l /usr/bin/env | while read f
   chmod a-x $file
 done
 
-# Remove rpaths from files
-chrpath --delete %{buildroot}%_libdir/obs-api/ruby/*/gems/sassc-*-x86_64-linux/lib/sassc/libsass.so
+# Remove rpaths from files (legacy)
+# if service bundle_gems ran with force_ruby_platform
```

Comment:
I really don't think we need to cover for this case. The service from now on will always run with `force_ruby_platform=true`...

Let's just remove this `chrpath` call and be done with it.